src/preview/font_match.rs

```diff
@@ -246,22 +230,17 @@ mod no_fontconfig {
 
     fn try_create_font_descriptor_from_face(path: PathBuf, face: &Face) -> Option<FontDescriptor> {
         let family_name = face.family_name();
-        if family_name.is_none() {
-            return None;
-        }
+        family_name.as_ref()?;
         let family_name = family_name.unwrap();
```

Comment:
I get a feeling this can also be simplified to:
```
let family_name = face.family_name()?;
```

Reply:
0861fa6

(TFW when you've already made this change a couple hours ago but haven't committed)